config/routes.rb

```diff
@@ -2,5 +2,9 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
-  # root "articles#index"
+  root "users#index"
+  resources :users, only: [:show] do
+    resources :posts, only: [:index, :show]
+  end
+
```

Comment:
- Good job done so far, but according to the project requirements, you are to add an `index` route to your application routes just like you did for the posts. Please fix this 🙏

Reply:
Ohh! thank you. I will do it instantly.

Reply:
@mckent05 [done](https://github.com/ZewdieMc/blog-app/pull/5/commits/418f34d5d77b09b4301d140cd3ebb3bd76688e2f)